### ISSUE TITLE
Pass -q to git reset --hard

### ISSUE THREAD

**Issue by dra27** [ocurrent/ocaml-ci#239]
Will suppress the many lines of "Checking out files" - errors will still be displayed.

**Comment by talex5** [comment]
I think it's actually coming from the pull: see #238.

**Comment by avsm** [comment]
fixed by #238

**Comment by dra27** [comment]
I don't know if it will still matter, but `git reset` will also display that if the working tree is hugely different

```
dra27@summer:~/opam-repository$ rm -rf packages && git reset --hard upstream/master
Checking out files: 100% (17364/17364), done.
HEAD is now at 698815dce8 Merge pull request #16998 from hhugo/jsoo-menhir
```
